app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryOwnedBy.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SelectUser from '../select_user/SelectUser';
+
+export default ExpandedStoryOwnedBy = ({ users, story, onEdit }) => (
+  <div className="Story__section">
+    <div className="Story__section-title">
+      {I18n.translate('activerecord.attributes.story.owned_by')}
+    </div>
+    <SelectUser
+      users={users}
+      userId={story._editing.ownedById}
+      onEdit={onEdit}
+    />
+  </div>
+);
+
+ExpandedStoryOwnedBy.PropTypes = {
+  users: PropTypes.array.isRequired,
+  story: PropTypes.object.isRequired,
+  story: PropTypes.shape({
```

Comment:
Is it necessary to duplicate the `story` entry here? I think you could just use the `shape` version, without the need to keep the other one (the one that is typing with `object`)

Reply:
I was thinking I should look to what kind `story` is, and your `shapes`! Thanks